diff --git a/src/configs/main.js b/src/configs/main.js
index e921b1b15eab13eb50d1c39f910b440ce5cdf5d9..ce052bea86f7cfef2ae23466c32096bb052e4a50 100644
--- a/src/configs/main.js
+++ b/src/configs/main.js
@@ -21,16 +21,14 @@ function isTSXSource(fileName) {
   return !!fileName && fileName.endsWith('.tsx');
 }
 
-// use `this.foo = bar` instead of `this.defineProperty('foo', ...)`
-const loose = true;
-
 const defaultPlugins = [
   [require('@babel/plugin-syntax-flow')],
   [require('babel-plugin-transform-flow-enums')],
   [require('@babel/plugin-transform-block-scoping')],
-  [require('@babel/plugin-proposal-class-properties'), {loose}],
-  [require('@babel/plugin-transform-private-methods'), {loose}],
-  [require('@babel/plugin-transform-private-property-in-object'), {loose}],
+  [require('@babel/plugin-transform-class-static-block')],
+  [require('@babel/plugin-transform-class-properties')],
+  [require('@babel/plugin-transform-private-methods')],
+  [require('@babel/plugin-transform-private-property-in-object')],
   [require('@babel/plugin-syntax-dynamic-import')],
   [require('@babel/plugin-syntax-export-default-from')],
   ...passthroughSyntaxPlugins,
