diff --git a/index.d.ts b/index.d.ts
index 0d936cf7ca70043f218170bbc446ae2e91816f46..4a60f5068307d85ee2cd2093b80f25c51d43e6c4 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -468,7 +468,8 @@ declare namespace chroma {
 
         gamma(g: number): this;
 
-        cache(use: boolean): boolean;
+        cache(use: boolean): this;
+        cache(): boolean;
 
         correctLightness(enable?: boolean): this;
 
